### ISSUE TITLE
Using Verbrauch widget from vis-2-widgets-energy causes blank screen

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.vis#680]
**Describe the bug**  
Using Verbrauch widget from vis-2-widgets-energy causes blank screen

**To Reproduce**  
1. Go to vis2 editor
2. Drag widget "Verbrauch" from vis2 energy section to vis view
3. Blank screen is shown -> error

**Expected behavior**  
Widget can be used in view

**Versions:**  
 - Adapter version: vis2 v2.0.23
 - vis-2-widgets-energy 0.1.10

**Screenshots & Logfiles**  
```
TypeError: jt is undefined
    Dr resizeObserver.js:64
    Vt sensorPool.js:60
    ee index.js:39
    dispose core.tsx:100
    componentWillUnmount core.tsx:82
    b5 main.8b8def71.js:1914
instrument.ts:129:35
TypeError: l is undefined
    s LegendModel.js:189
    G util.js:228
    _updateData LegendModel.js:181
    optionUpdated LegendModel.js:119
    f Global.js:386
    A util.js:205
    f Global.js:308
    topologicalTravel component.js:122
    _mergeOption Global.js:287
    m1 Global.js:777
    _resetOption Global.js:199
    setOption Global.js:172
    setOption echarts.js:418
    updateEChartsOption core.tsx:160
    renderNewEcharts core.tsx:116
    componentDidMount core.tsx:38
    Tt main.8b8def71.js:1914
    ct main.8b8def71.js:1914
    ot main.8b8def71.js:1914
    yc main.8b8def71.js:1914
    $r main.8b8def71.js:1914
    Fo main.8b8def71.js:1914
    k3 main.8b8def71.js:1912
    c3 main.8b8def71.js:1914
    c3 main.8b8def71.js:1914
    b3 main.8b8def71.js:1914
    enqueueSetState main.8b8def71.js:1912
    setState main.8b8def71.js:1940
    setStateAsync Runtime.jsx:95
    setStateAsync Runtime.jsx:94
    it App.jsx:946
    De 360.e76577e1.chunk.js:33
    De 360.e76577e1.chunk.js:33
    it App.jsx:918
    it App.jsx:441
    De 360.e76577e1.chunk.js:33
    De 360.e76577e1.chunk.js:33
    it App.jsx:386
    drop App.jsx:146
    drop 993.be355181.chunk.js:805
    B1 993.be355181.chunk.js:805
    2432/$/</< 993.be355181.chunk.js:805
    2432/$/< 993.be355181.chunk.js:805
    _1 993.be355181.chunk.js:805
    handleTopDrop 993.be355181.chunk.js:805
    addEventListeners 993.be355181.chunk.js:805
    setup 993.be355181.chunk.js:805
    handleRefCountChange 993.be355181.chunk.js:805
    Z 993.be355181.chunk.js:805
    addTarget 993.be355181.chunk.js:805
    T 993.be355181.chunk.js:805
    d 993.be355181.chunk.js:805
    O2 main.8b8def71.js:1914
    Tt main.8b8def71.js:1914
    ct main.8b8def71.js:1914
    ot main.8b8def71.js:1914
    yc main.8b8def71.js:1914
    $r main.8b8def71.js:1914
    io main.8b8def71.js:1914
    x main.8b8def71.js:1948
    w main.8b8def71.js:1948
    41196 main.8b8def71.js:1948
    de main.8b8def71.js:1950
    72851 main.8b8def71.js:1948
    de main.8b8def71.js:1950
    57799 main.8b8def71.js:1907
    de main.8b8def71.js:1950
    73961 main.8b8def71.js:1914
    de main.8b8def71.js:1950
    I main.8b8def71.js:1950
    R main.8b8def71.js:1951
    de main.8b8def71.js:1950
    24470 main.8b8def71.js:1914
    de main.8b8def71.js:1950
    I main.8b8def71.js:1950
    P main.8b8def71.js:1951
    de main.8b8def71.js:1950
    98360 360.e76577e1.chunk.js:10
    de main.8b8def71.js:1950
    promise callback*72063 main.8b8def71.js:1
    de main.8b8def71.js:1950
    <anonymous> main.8b8def71.js:1953
    <anonymous> main.8b8def71.js:1953
instrument.ts:129:35
Uncaught TypeError: l is undefined
    s LegendModel.js:189
    G util.js:228
    _updateData LegendModel.js:181
    optionUpdated LegendModel.js:119
    f Global.js:386
    A util.js:205
    f Global.js:308
    topologicalTravel component.js:122
    _mergeOption Global.js:287
    m1 Global.js:777
    _resetOption Global.js:199
    setOption Global.js:172
    setOption echarts.js:418
    updateEChartsOption core.tsx:160
    renderNewEcharts core.tsx:116
    componentDidMount core.tsx:38
    Tt main.8b8def71.js:1914
    ct main.8b8def71.js:1914
    ot main.8b8def71.js:1914
    yc main.8b8def71.js:1914
    $r main.8b8def71.js:1914
    Fo main.8b8def71.js:1914
    k3 main.8b8def71.js:1912
    c3 main.8b8def71.js:1914
    c3 main.8b8def71.js:1914
    b3 main.8b8def71.js:1914
    enqueueSetState main.8b8def71.js:1912
    setState main.8b8def71.js:1940
    setStateAsync Runtime.jsx:95
    setStateAsync Runtime.jsx:94
    it App.jsx:946
    De 360.e76577e1.chunk.js:33
    De 360.e76577e1.chunk.js:33
    it App.jsx:918
    it App.jsx:441
    De 360.e76577e1.chunk.js:33
    De 360.e76577e1.chunk.js:33
    it App.jsx:386
    drop App.jsx:146
    drop 993.be355181.chunk.js:805
    B1 993.be355181.chunk.js:805
    2432/$/</< 993.be355181.chunk.js:805
    2432/$/< 993.be355181.chunk.js:805
    _1 993.be355181.chunk.js:805
    handleTopDrop 993.be355181.chunk.js:805
    addEventListeners 993.be355181.chunk.js:805
    setup 993.be355181.chunk.js:805
    handleRefCountChange 993.be355181.chunk.js:805
    Z 993.be355181.chunk.js:805
    addTarget 993.be355181.chunk.js:805
    T 993.be355181.chunk.js:805
    d 993.be355181.chunk.js:805
    O2 main.8b8def71.js:1914
    Tt main.8b8def71.js:1914
    ct main.8b8def71.js:1914
    ot main.8b8def71.js:1914
    yc main.8b8def71.js:1914
    $r main.8b8def71.js:1914
    io main.8b8def71.js:1914
    x main.8b8def71.js:1948
    w main.8b8def71.js:1948
    41196 main.8b8def71.js:1948
    de main.8b8def71.js:1950
    72851 main.8b8def71.js:1948
    de main.8b8def71.js:1950
    57799 main.8b8def71.js:1907
    de main.8b8def71.js:1950
    73961 main.8b8def71.js:1914
    de main.8b8def71.js:1950
    I main.8b8def71.js:1950
    R main.8b8def71.js:1951
    de main.8b8def71.js:1950
    24470 main.8b8def71.js:1914
    de main.8b8def71.js:1950
    I main.8b8def71.js:1950
    P main.8b8def71.js:1951
    de main.8b8def71.js:1950
    98360 360.e76577e1.chunk.js:10
    de main.8b8def71.js:1950
    promise callback*72063 main.8b8def71.js:1
    de main.8b8def71.js:1950
    <anonymous> main.8b8def71.js:1953
    <anonymous> main.8b8def71.js:1953

```

**Comment by GermanBluefox** [comment]
Please retest with new energy version

**Comment by Feuer-sturm** [comment]
Issue is fixed with energy adapter version 0.1.11
Thanks.
